pkg/sql/vars.go

```diff
@@ -2389,6 +2389,21 @@ var varGen = map[string]sessionVar{
 		},
 		GlobalDefault: globalFalse,
 	},
+	`disable_split_limited_scan_cost_check`: {
```

Comment:
I should probably prefix this with `optimizer_` like we do with other session settings.